build.gradle

```diff
@@ -214,6 +215,8 @@ subprojects {
                 netty_proxy_handler: "io.netty:netty-handler-proxy:${nettyVersion}",
                 netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:2.0.7.Final',
 
+                conscrypt: 'org.conscrypt:conscrypt-openjdk-uber:1.0.0',
```

Comment:
intended?

Reply:
Yes. Jetty ALPN doesn't exist in Java 9. So for OkHttp TLS testing it will use Conscrypt instead.